src/Generator/Generators/CLI/CLIHeadersTemplate.cs

```diff
@@ -137,6 +137,9 @@ public void GenerateForwardRefs()
             var typeReferences = typeReferenceCollector.TypeReferences;
             var @namespace = ConvertForwardReferencesToNamespaces(typeReferences);
 
+            foreach (var cls in @namespace.Classes)
+                cls.IsStruct = ExtensionMethods.CheckClassIsStructible(cls, Driver);
```

Comment:
Ok. Will do.
But how do I make sure that pass is applied to say Basic.h in the tests!?